specificatie/openapi.yaml

```diff
@@ -627,18 +627,18 @@ components:
         - $ref: '#/components/schemas/ZoekResultaat'
         - properties:
             _links:
-              $ref: '#/components/schemas/ZoekResultaat_links'
-    ZoekResultaat_links:
+              $ref: '#/components/schemas/ZoekResultaatLinks'
+    ZoekResultaatLinks:
       type: object
       properties:
         adres:
-          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.1.0/api-specificatie/common.yaml#/components/schemas/HalLink'
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.3.0/api-specificatie/common.yaml#/components/schemas/HalLink'
     ZoekResultaatHalCollectie:
       type: object
       description: "Resultaten als lijst."
       properties:
         _links:
-          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.1.0/api-specificatie/common.yaml#/components/schemas/HalPaginationLinksMetLast'
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.3.0/api-specificatie/common.yaml#/components/schemas/HalPaginationLinksMetLast'
         _embedded:
```

Comment:
Het is mij niet eerder opgevallen, maar dit zorgt ervoor dat er een class met de naam _embedded wordt gegenereerd